pkg/util/config/config.go

```diff
@@ -103,6 +103,7 @@ type Config struct {
 	TeamAPIRoleConfiguration map[string]string `name:"team_api_role_configuration" default:"log_statement:all"`
 	PodTerminateGracePeriod  time.Duration     `name:"pod_terminate_grace_period" default:"5m"`
 	ProtectedRoles           []string          `name:"protected_role_names" default:"admin"`
+	PostgresAdminTeams       []string          `name:"postgres_admin_teams" default:"postgres_admin"`
```

Comment:
Please make this an empty default, don't want people to create a team called postgres_admin and suddenly get super power.